target-config.js~

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Is this a file generated by sublime? If so, please add it (and other files of type .js~) to your .gitignore